crypter_test.go

```diff
@@ -33,6 +37,26 @@ var rsaTestKey, _ = rsa.GenerateKey(rand.Reader, 2048)
 var ecTestKey256, _ = ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
 var ecTestKey384, _ = ecdsa.GenerateKey(elliptic.P384(), rand.Reader)
 var ecTestKey521, _ = ecdsa.GenerateKey(elliptic.P521(), rand.Reader)
+var x509TestCertificate = func() *x509.Certificate {
+	serialNumberLimit := new(big.Int).Lsh(big.NewInt(1), 128)
+	serialNumber, _ := rand.Int(rand.Reader, serialNumberLimit)
+	t := x509.Certificate{
+		SerialNumber: serialNumber,
+		Subject: pkix.Name{
+			Organization: []string{"Acme Co"},
+		},
+		NotBefore: time.Now().Round(time.Second),
+		NotAfter: time.Now().Add(time.Hour).Round(time.Second),
+		KeyUsage:              x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature,
+		ExtKeyUsage:           []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth},
+		BasicConstraintsValid: true,
+	}
+	t.IsCA = true
+	t.KeyUsage |= x509.KeyUsageCertSign
+	der, _ := x509.CreateCertificate(rand.Reader, &t, &t, &rsaTestKey.PublicKey, rsaTestKey)
+	c, _ := x509.ParseCertificate(der)
```

Comment:
No need to create a certificate on the fly, just throw an hard-coded one in there. 

Here's one we can use for testing:
```
MIIDYTCCAkmgAwIBAgIJAKNePtFE5+2PMA0GCSqGSIb3DQEBCwUAMD4xCzAJBgNV
BAYTAlVTMQswCQYDVQQIEwJDQTETMBEGA1UEChMKZ2hvc3R1bm5lbDENMAsGA1UE
CxMEcm9vdDAeFw0xNjAyMjIxOTE5NTNaFw0xODExMTgxOTE5NTNaMFExCzAJBgNV
BAYTAlVTMQswCQYDVQQIEwJDQTETMBEGA1UEChMKZ2hvc3R1bm5lbDEPMA0GA1UE
CxMGY2xpZW50MQ8wDQYDVQQDEwZjbGllbnQwggEiMA0GCSqGSIb3DQEBAQUAA4IB
DwAwggEKAoIBAQDARiaDcWQc0TRoncxdF8M/l4OPmor4nVf7e1JLOnMaJiwrIvdG
tRMnkM+wJKtIlSg69t2kN3UsuFA6MOhxHyFYxxX69EfV/foMqDHG/+2BwhCd1iQ6
8XoKh5Plu+ItNv1nJaTfBtw2XcaR2m3oiiwbl+6eYNWWDwiNKsj9R+HN8YDPWq4P
UYf9SLycX85ZIqjNv4mtVdLjtFvksH3kD1aUNTO29dTtAyVEPyeYDbkTcDzU6oPK
sklRO4YOJyr5y9BgTVXBRr2GTbeVBF06n94zGrKc1Y54tDLYwXgkcP6MAP9/QiWv
xHPPdbsGB8Y67AQSrydj6qJmq+kA4s/AwYlRAgMBAAGjTzBNMB0GA1UdJQQWMBQG
CCsGAQUFBwMCBggrBgEFBQcDATAsBgNVHREEJTAjhwR/AAABhxAAAAAAAAAAAAAA
AAAAAAABgglsb2NhbGhvc3QwDQYJKoZIhvcNAQELBQADggEBAJkbNP6zGZhOhuLU
+AuNHbs2usiJwyM1nZu8yIXtL/NdxjJCoImSVtiCxBIFoQaGVlyDaRZyr+HtlMEf
a62Go8DrJXd0ZsPto+sByOKQPafbV74vrgVhRUWBTXmVqzfYcEE9XYxRXrJofzMz
7nYl/zGa/pe+9cB68+s9Gi5zvtmP9c0nG2JLr5FakD3lCCfCA6tCDiCGfYcIR+f+
v9K2vTzGRqQu0W+u3eEcEi2CVlIml09vBcRKNSC9j3qQtQDYAfHsAX24kXpPEKDC
s4Rmeph13uR3dhFxbfbeQFIrDvZB4GOI19zV3hqYd8HYqY49WSaOzwGK8hAm+OtJ
o23Dhfs=
```